guides/common/modules/proc_renaming-server.adoc

```diff
@@ -2,36 +2,34 @@
 = Renaming {ProjectServer}
 
 The host name of {ProjectServer} is used by {ProjectServer} components, all {SmartProxyServers}, and hosts registered to it for communication.
-This procedure ensures that you update all references to the new host name.
+This procedure ensures that in addition to renaming {ProjectServer}, you also update all references to point to the new host name.
 
-If you use external authentication, you must reconfigure {ProjectServer} for external authentication after you run the `{project-change-hostname}` script.
-The `{project-change-hostname}` script breaks external authentication for {ProjectServer}.
-ifndef::orcharhino[]
-For more information about configuring external authentication, see {InstallingServerDocURL}Configuring_External_Authentication_{project-context}[Configuring External Authentication] in _{InstallingServerDocTitle}_.
-endif::[]
-
-If you use virt-who, you must update the virt-who configuration files with the new host name after you run the `{project-change-hostname}` script.
-ifdef::satellite[]
-For more information, see {ConfiguringVMSubscriptionsDocURL}troubleshooting-virt-who#modifying-virt-who-configuration_vm-subs-satellite[Modifying a virt-who Configuration] in _{ConfiguringVMSubscriptionsDocTitle}_.
-endif::[]
+[WARNING]
+====
+Renaming your {ProjectServer} host shuts down all {ProjectServer} services on that host.
+The services restart after the renaming is complete.
+====
 
 .Prerequisites
-* Both the `hostname` and `hostname -f` commands must return the FQDN of {ProjectServer} or the `{project-change-hostname}` script will fail to complete.
-If the `hostname` command returns the shortname of {ProjectServer} instead of the FQDN, use `hostnamectl set-hostname _old_fqdn_` to set the old FQDN correctly before attempting to use the `{project-change-hostname}` script.
-
-* Perform a backup of {ProjectServer} before changing a host name.
-If the renaming process is not successful, you must restore it from a backup.
+* Back up your {ProjectServer} before changing its host name.
+If you fail to successfully rename it, restore it from the backup.
 For more information, see xref:backing-up-{project-context}-server-and-{smart-proxy-context}_{context}[].
 
-* Optional: If {ProjectServer} has a custom SSL certificate installed, a new certificate must be obtained for the host's new name.
+* Run the `hostname` and `hostname -f` commands on {ProjectServer}. 
+Both commands must return the FQDN of {ProjectServer} or the `{project-change-hostname}` script will fail to complete.
++
+If the `hostname` command returns the shortname of {ProjectServer} instead of the FQDN, use `hostnamectl set-hostname _old_fqdn_` to set the old FQDN correctly before using the `{project-change-hostname}` script.
```

Comment:
```suggestion
If the `hostname` command returns the shortname of {ProjectServer} instead of the FQDN, use `hostnamectl set-hostname _My_Old_FQDN_` to set the old FQDN correctly before using the `{project-change-hostname}` script.
```